packages/docusaurus-plugin-sitemap/src/createSitemap.ts

```diff
@@ -0,0 +1,36 @@
+/**
+ * Copyright (c) 2017-present, Facebook, Inc.
+ *
+ * This source code is licensed under the MIT license found in the
+ * LICENSE file in the root directory of this source tree.
+ */
+
+import sitemap, {SitemapItemOptions} from 'sitemap';
+import {PluginOptions} from './types';
+import {DocusaurusConfig} from '@docusaurus/types';
+
+export default function createSitemap(
+  siteConfig: DocusaurusConfig,
```

Comment:
any reason we change this function signature? any existing usages affected?

Reply:
No one is affected - this is internal for the plugin. Without changing the signature I would have to introduce an ad hoc type for the passed object which looks quite ugly.